apps/redi-talent-pool/src/components/organisms/company-profile-editables/EditableJobPostings.tsx

```diff
@@ -185,6 +157,100 @@ interface ModalFormProps {
   setIsEditing: (boolean) => void
 }
 
+function JobListingCards({ jobListings, startEditing }) {
+  const queryClient = useQueryClient()
+
+  const renderTimestamp = (expiresAt) =>
+    `Expires ${formatDistance(new Date(expiresAt), new Date(), {
+      addSuffix: true,
+    })}`
```

Comment:
It's a very nitpick but usually defining these kind of util or render functions inside a component causes this function to be created and referenced every time this component rerenders. So it's a better approach to define these outside the component, especially when they don't depend on any state or props of the component.

Reply:
Agreed. Refactored.